src/dmtcp_restart.cpp

```diff
@@ -404,8 +404,42 @@ getMtcpArgs()
   return mtcpArgs;
 }
 
+vector<string>
+getMtcpEnviron(RestoreTarget *restoreTarget)
```

Comment:
I suggest, everywhere:
Envrion -> Evironment

The 'main' routine calls this 'envp', and in most other cases, it is called 'environment'.  It's confusing to add a third term.